src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -147,6 +147,9 @@ public class LibraryServiceSettings extends ClientSettings {
       .build();
 
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
Sgtm. We might want to do something similar to the license matching, which allows the year number to be dynamic.